modules/tutorials-for-data-scientists.adoc

```diff
@@ -83,17 +83,19 @@ ifndef::upstream[]
 | Importing Pachyderm Beginner Tutorial Notebook
 | Load Pachyderm's beginner tutorial notebook and learn about Pachyderm's main concepts such as data repositories, pipelines, and using the pachctl CLI from your cells.
 
-| Installing and verifying the NVIDIA GPU Add-on
```

Comment:
I initially took this out, but then found https://issues.redhat.com/browse/RHODS-12942 so I will make those updates separately/next.